github_actions/lib/dependabot/github_actions/update_checker.rb

```diff
@@ -328,6 +328,11 @@ def find_container_branch(sha)
           branches_including_ref.first
         end
       end
+
+      sig { params(sha: String).returns(String) }
+      def git_branches_containing(sha)
```

Comment:
Is there a reason this needs to be extracted as a function? The type checking adds little to no value here IMO... Again, the code is more readable if the command is inlined...